go.mod

```diff
@@ -2,4 +2,17 @@ module github.com/timberio/http_test_server
 
 go 1.13
 
-require github.com/urfave/cli v1.22.1
```

Comment:
I switched to `viper`/`cobra` given `urfave/cli` is not as well maintained these days and I wanted easier out-of-the-box configuration (like handling environment variables for flags).